site/user-documentation/nuitka-package-config.rst

```diff
@@ -662,14 +662,16 @@ To check what OS is selected, we got these.
 Compilation modes
 -----------------
 
-|  ``standalone``: ``True`` if standalone mode is activated with
-   ``--mode=standalone``, ``--mode=onefile`` or ``--mode=app``
-|  ``onefile``: ``True`` if onefile mode is activated with
-   ``--mode=onefile`` or with ``--mode=app`` on non-macOS
-|  ``module_mode``: ``True`` if module mode is activated with
-   ``--mode=module``
-|  ``deployment``: ``True`` if deployment mode is activated with
-   ``--deployment``
+|  ``standalone``: ``True`` when creating distributable applications that
+   include all dependencies. Activated with ``--standalone``, ``--onefile``,
```

Comment:
Please use the --mode syntax, as these ones are going away with 2.8 from at least help output and not to be used anywhere at all anymore.